scintellometry/io/arochime.py

```diff
@@ -15,6 +15,7 @@
 
 from . import SequentialFile, header_defaults
 
+import pfb
```

Comment:
This import will mean that effectively `pfb` becomes a dependency for `scintellometry`, since one cannot access the `AROCHIME` readers without importing it. I think instead we should either make it part of `scintellometry`, or do a local import.